Exit node maintenance

DerfOh opened the pull request:
## Reopening from: Exit node maintenance #26

I added the remaining exit nodes from the `locations-v3.xml file`. I also commented out any that would refuse to connect. If needed I can open an issue for this, I'm unsure why the connection does not complete.

Also I added a list of exit nodes that I extracted from the `locations-v3.xml` file to `ProXPN Exit Nodes.md`

This is my first (of few) pull request in a very long time so feel free to reach out to me if you need me to do anything additional.
_Edit: also rebased_

MattSurabian commented:
Thanks for this! Do you mind re-basing it down to a single commit? 

Interesting that they've closed some of these nodes. Wonder what that's about...

DerfOh commented:
Rebase complete. Sorry for the wait, first time doing that in a while.

DerfOh committed:
Added additional exit nodes for proXPN

Added: Toronto, Montreal, HongKong, Sydne, Frankfurt, NewDelhi, Tokyo,
Bangkok, Hafnarfjordur, Zurich, and Paris.
Also removed double entry for Stockholm exit node.

Added list of known exit nodes. Commented out exit nodes that are
disfunctional.

MattSurabian commented:
Sorry for the delay here, and thanks for rebasing! Wish I had some more context for why these exit nodes have gone dark... regardless I see that they have. Thanks for noticing and addressing this!
